pandas/core/groupby.py

```diff
@@ -2322,9 +2326,16 @@ def _aggregate(self, result, counts, values, comp_ids, agg_func,
             for i, chunk in enumerate(values.transpose(2, 0, 1)):
 
                 chunk = chunk.squeeze()
-                agg_func(result[:, :, i], counts, chunk, comp_ids)
+                if min_count is None:
```

Comment:
I would *always* pass it to cython as an int. it can be -1 if its ``None`` somehow

Reply:
I mainly did this to avoid having to add an unused `min_count` keyword to all the `group_*` methods. If I added `min_count` to each of those, then I'd also need to raise if the non-default value is passed. I'll add a comment here that it's only expected for `count` and `prod` though.

Reply:
I would just add it and assert == -1. these are all internal routines anyhow. an if is very ugly and error prone here. better to make them have the same uniform signature.

Reply:
Ah, ok that sounds easy enough.